frontend/cypress/perf/common.ts

```diff
@@ -0,0 +1,41 @@
+export const reportFilePath = 'logs/performance.txt';
```

Comment:
```suggestion
export const reportFilePath = 'cypress/results/performance.txt';
```
I would include performance results within cypress folder